packages/cli/cmd/application/main.go

```diff
@@ -77,15 +101,20 @@ func BuildCommandDocsForHugo(cmd *cobra.Command, dir string) error {
 
 func buildRootCmd() *cobra.Command {
 	vars := mainVars{}
+	formatVars := formatVars{}
 	cmd := &cobra.Command{
 		Use:   "agc",
 		Short: shortDescription,
 		Example: `
   Displays the help menu for the specified sub-command.
   /code $ agc account --help`,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			format.SetFormatter(format.FormatterType(vars.format))
 			setLoggingLevel()
+			opts, err := newFormatOpts(formatVars)
```

Comment:
- It doesn't seem like error affect our execution flow in any way here. It means we can make this code simpler:
```go
PersistentPreRun: func(cmd *cobra.Command, args []string) {
  setLoggingLevel()
  setFormatter(formatVars)
  checkCliVersion()
```
In my opinion this way the lambda looks cleaner without loosing any functionality.

Creation of a client and logging of the error can happen inside `setFormatter` method itself.

```go
func setFormatter(formatVars formatVars) {
	configClient, err := config.NewConfigClient()
	if err != nil {
		log.Error().Err(err)
		return
	}
...

}
```

In order to make it testable, you can use a factory method for client creation.

```go
var newConfigClient = config.NewConfigClient
```